Lib/imaplib.py

```diff
@@ -1166,7 +1166,7 @@ def _match(self, cre, s):
         self.mo = cre.match(s)
         if __debug__:
             if self.mo is not None and self.debug >= 5:
-                self._mesg("\tmatched r'%r' => %r" % (cre.pattern, self.mo.groups()))
+                self._mesg("\tmatched %r => %r" % (cre.pattern, self.mo.groups()))
```

Comment:
I think what this actually wanted to be was `r'%s' % cre.pattern.replace('\\\\', '\\')`, but of course that doesn't work in all cases.  It would be nice if we had a codec that produced raw string representation (it would have to fall back to repr for cases that can't be represented as raw strings).

Reply:
I will think about it.